tests/opwen_email_server/utils/test_email_parser.py

```diff
@@ -13,6 +14,13 @@
     'files', 'opwen_email_server', 'utils', 'test_email_parser'))
 
 
+def _given_test_image_base64(size):
+    b64_images_file = join(TEST_DATA_DIRECTORY, 'image_base64.json')
```

Comment:
Nit: given that the file contains multiple images, should it be called 'images_base64.json'?